example/app.js

```diff
@@ -1,46 +1,51 @@
+import React from 'react'
 import PropTypes from 'prop-types'
+import DOM from 'react-dom-factories'
```

Comment:
I think we can get rid of this. This was preferable solution in CoffeeScript, now it's obsolete. Can we use JSX everywhere instead?